app/views/install/compose.phtml

```diff
@@ -153,6 +155,50 @@ services:
       - _APP_DB_PASS
       - _APP_USAGE_STATS
 
+  appwrite-executor:
+    image: <?php echo $organization; ?>/<?php echo $image; ?>:<?php echo $version."\n"; ?>
+    entrypoint: executor
+    container_name: appwrite-executor
+    stop_signal: SIGINT
+    networks:
+      appwrite:
+      runtimes:
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - appwrite-functions:/storage/functions:rw
+      - /tmp:/tmp:rw
```

Comment:
do we need this?

Reply:
Yes, but we have a future task for refactoring all the functions mounts names.